apps/core-v2-testnet/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  env: {
+    browser: false,
+    es2021: true,
+    mocha: true,
+    node: true,
+  },
+  plugins: ["@typescript-eslint"],
+  extends: [
+    "standard",
+    "plugin:prettier/recommended",
```

Comment:
We can remove prettier here, since elementfi/eslint-config includes the prettier plugin already.